unfoldingWord/translationCore
# Created my projects FAB

### This pull request addresses:

Created my projects FAB #1980

### How to test this pull request:

- Go to the `My Projects` section and click on the FAB on the bottom right. 
- Should open other FAB buttons which at this point shouldn't be functional.
- Each button should have label cards on the left as in the mockups


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2024)
